opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -19,16 +19,13 @@
 The `Meter` class is used to construct `Metric` s to record raw statistics
 as well as metrics with predefined aggregation.
 
+`Meter` s can be obtained via the `MeterProvider`, corresponding to the name
+of the instrumenting library and (optionally) a version.
+
 See the `metrics api`_ spec for terminology and context clarification.
 
 .. _metrics api:
-    https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-metrics.md
-
-.. versionadded:: 0.1.0
-.. versionchanged:: 0.5.0
-    ``meter_provider`` was replaced by `get_meter_provider`,
-    ``set_preferred_meter_provider_implementation`` was replaced by
-    `set_meter_provider`.
```

Comment:
It may well make sense to lose the version comments until we're out of beta, but why do it in this PR?

Reply:
CHANGELOG.md contains a lot more information than the version comments (some missed API changes related to metrics). Instead of updating in multiple places, we can just refer to that file.